airbyte-integrations/connectors/source-mixpanel/source_mixpanel/schemas/export.json

```diff
@@ -248,6 +245,21 @@
     },
     "mp_processing_time_ms": {
       "type": ["null", "string"]
+    },
+    "region": {
```

Comment:
why do we need to update the schema as part of the PR?

Reply:
expected records test failed, I added record with test data from simple data set and faced that our schema didn't have those fields so I updated export schema.